catroid/src/main/java/org/catrobat/catroid/ui/fragment/CategoryBricksFactory.java

```diff
@@ -495,6 +500,10 @@
 				BrickValues.REPLACE_ITEM_IN_USERLIST_INDEX));
 		dataBrickList.add(new AskBrick(context.getString(R.string.brick_ask_default_question)));
 		dataBrickList.add(new AskSpeechBrick(context.getString(R.string.brick_ask_speech_default_question)));
+
+		if (BuildConfig.DEBUG) {
```

Comment:
we always use feature toggles for this kind of thing. because then we can enable / disable features like this from the gradle file instead of digging through the code and locating this block here and the one above.